frappe/www/login.py

```diff
@@ -143,3 +145,49 @@ def login_via_token(login_token):
 	redirect_post_login(
 		desk_user=frappe.db.get_value("User", frappe.session.user, "user_type") == "System User"
 	)
+
+
+@frappe.whitelist(allow_guest=True)
+def send_login_link(email: str, subject: str | None = None):
+	if not frappe.db.exists("User", email):
```

Comment:
This is only safe on `develop` since yesterday, so we should take care when backporting.